src/applications/vaos/services/appointment/index.js

```diff
@@ -250,27 +222,18 @@ export function getVARClinicId(appointment) {
  * @returns {string} The location id where the VA appointment is located
  */
 export function getVAAppointmentLocationId(appointment) {
```

Comment:
I'd like to remove this helper as well, but there are a lot of uses and this PR is already a decent size.